src/components/project/DescriptionSection/DescriptionSection.tsx

```diff
@@ -0,0 +1,20 @@
+import { css } from '@emotion/react';
+
+import { colors } from '~/styles/constants';
+
+export default function HeaderSection() {
+  return (
+    <div css={descriptionCss}>
+      <div>디프만에서 탄생한 개성만점</div>
+      <div>서비스들을 소개할게요</div>
+    </div>
```

Comment:
```suggestion
    <header css={descriptionCss}>
      <h2>디프만에서 탄생한 개성만점<br/>서비스들을 소개할게요</h2>
    </div>
```

요렇게 시맨틱 태그를 쓰는게 어떨까요!? (꼭은 아닙니당)

Reply:
https://github.com/depromeet/www.depromeet.com/pull/105/commits/dff28489efd102456fb0647b92f5504ac7497a06 - 넘 좋습니다 ㅋㅋㅋ 반영하였습니다.